### PR TITLE
Login User Interface ✅✅

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
## In this PR

- [ ]  I have created the basic UI for the login page.✔✔

- [ ] Store the username to the database when a user logged in **✔✔**

- [ ] Give the path of the home page when the user clicks the login button.✔✔

### CODE REVIEW

File: app/javascript/routes/index.jsx [hunk 1]
```diff
@@ -19,6 +22,8 @@ export default (
         <Route path="/reserve" element={<ReservationPage />} />
         <Route path="/myreservations" element={<MyReservationsPage />} />
         <Route path="/addcar" element={<AddCarPage />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/initial" element={<Body />} />
```
Comment (roniy68) [inline app/javascript/routes/index.jsx:26]: Login Should be out of the nested route

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
+
+    resource '*',
+             headers: :any,
+             methods: %i[get post put patch delete options head]
+  end
+end
```
Comment (roniy68) [inline config/initializers/cors.rb:25]: Remove the cors.rb .. We don't need outside API fetch

File: config/routes.rb [hunk 3]
```diff
@@ -6,6 +6,8 @@
       resources :cars, only: %i[index create show destroy]
       resources :reservations, only: %i[index create show destroy]
       get '/users/:user_id/reservations', to: 'reservations#user_reservations'
+      post 'users/signin', to: 'sessions#create'
+      get 'users/index'
```
Comment (roniy68) [inline config/routes.rb:10]: 👍 SEssion controller Handled

**Review by roniy68 — CHANGES_REQUESTED** [review]
# Hello @Talha-Hanif5666 

Almost Done. But Some Required changes need to be  resolved. Nonetheless great work.
 



# :heavy_check_mark: Approved :heavy_check_mark: 

It was a pleasure reviewing your project I have learned a lot from your codebase as well :raised_hands: . Keep doing the super work that you just did here! :clap: 


Cheers and Happy coding! :rocket:

File: config/routes.rb (hunk 3 above)
Comment (Talha-Hanif5666) [reply config/routes.rb:10]: @roniy68 Thank you for your review, I have updated all changes. Please have a look!

**Pull request merged**